noxfile.py

```diff
@@ -221,6 +222,63 @@ def upload_wheel(session: nox.Session) -> None:
     session.run("twine", "upload", "dist/*", external=True)
 
 
+@nox.session
+def bundle_app(session: nox.Session) -> None:
+    """Bundle a standalone executable."""
+    session.install(
+        "jinja2",
+        "lxml",
+        "pygments",
+        "pyinstaller",
+    )
+    session.install("-e", ".")
```

Comment:
Is it necessary to install all these modules? I would guess that `install(".", "pyinstaller")` would be sufficient.

Reply:
If I don't use the wheel I need to install all needed modules. On my computer, e.g. jinja isn't installed.

Reply:
The same as for the tests.